src/converter/meta.py

```diff
@@ -11,14 +11,14 @@ def convert(pages: List[Page]) -> dict:
                 "artboards": {
                     artboard.do_objectID: {"name": artboard.name}
                     for artboard in page.layers
-                    if isinstance(artboard, Artboard) or isinstance(artboard, SymbolMaster)
+                    if isinstance(artboard, Frame) or isinstance(artboard, SymbolMaster)
                 },
             }
             for page in pages
         },
-        "version": 144,
+        "version": 156,
         "compatibilityVersion": 99,
-        "coeditCompatibilityVersion": 143,
+        "coeditCompatibilityVersion": 156,
```

Comment:
Support for frames is a breaking change, and so we're bumping the Sketch model and coedit compatibility versions to ensure the correct warnings are displayed when trying to open these documents in older versions of the Sketch Mac app.